pkg/storaged/mdraids-panel.jsx

```diff
@@ -0,0 +1,132 @@
+                                    { value: "raid6",  Title: _("RAID 6 (Double Distributed Parity)") },
+                                    { value: "raid10", Title: _("RAID 10 (Stripe of Mirrors)") }
+                                ]
+                              },
+                              { SelectOne: "chunk",
+                                Title: _("Chunk Size"),
+                                Options: [
+                                    { value: "4", Title: _("4 KiB") },
+                                    { value: "8", Title: _("8 KiB") },
+                                    { value: "16", Title: _("16 KiB") },
+                                    { value: "32", Title: _("32 KiB") },
+                                    { value: "64", Title: _("64 KiB") },
+                                    { value: "128", Title: _("128 KiB") },
+                                    { value: "512", Title: _("512 KiB"), selected: true },
+                                    { value: "1024", Title: _("1 MiB") },
+                                    { value: "2048", Title: _("2 MiB") }
+                                ],
+                                visible: function (vals) {
+                                    return vals.level != "raid1";
+                                }
+                              },
+                              { SelectMany: "disks",
+                                Title: _("Disks"),
+                                Options: get_available_spaces(client).map(available_space_to_option),
+                                EmptyWarning: _("No disks are available."),
+                                validate: function (disks, vals) {
+                                    var disks_needed = vals.level == "raid6"? 4 : 2;
+                                    if (disks.length < disks_needed)
+                                        return cockpit.format(_("At least $0 disks are needed."),
+                                                              disks_needed);
+                                }
+                              }
+                          ],
+                          Action: {
+                              Title: _("Create"),
+                              action: function (vals) {
+                                  return prepare_available_spaces(client, vals.disks).then(function () {
+                                      var paths = Array.prototype.slice.call(arguments);
+                                      return client.manager.MDRaidCreate(paths, vals.level,
+                                                                         vals.name, (vals.chunk || 0) * 1024,
+                                                                         { });
+                                  });
+                              }
+                          }
+            });
+        }
+
+        function cmp_mdraid(path_a, path_b) {
+            // TODO - ignore host part
```

Comment:
We don't have that function in the existing code. But this would at most be a presentational issue with an unusual sorting order, right?

Reply:
We do, in `overview.js`, no?  Do I misunderstand?